editor/src/clj/editor/welcome.clj

```diff
@@ -408,11 +408,20 @@
           (.setFixedCellSize 56.0)
           (ui/items! (recent-projects prefs))
           (ui/cell-factory!
-            (fn [recent-project]
-              {:graphic (make-recent-project-entry recent-project recent-projects-list prefs localization)}))
+           (fn [recent-project]
```

Comment:
Style: We indent with two spaces when the arguments are on a separate line from the function:
```clj
(ui/cell-factory!
  (fn [recent-project]
    ,,,)
```